File: kotlin/lib/src/main/kotlin/models/SqsConfigOut.kt
// This file is @generated
package com.svix.kotlin.models

import kotlinx.serialization.Serializable

@Serializable
data class SqsConfigOut(
    val queueUrl: String,
    val region: String,
    val accessKeyId: String? = null,
    val roleArn: String? = null,
    val externalId: String? = null,
    val endpointUrl: String? = null,
)

File: python/svix/models/sqs_config_out.py
# this file is @generated
import typing as t

from .common import BaseModel


class SqsConfigOut(BaseModel):
    queue_url: str

    region: str

    access_key_id: t.Optional[str] = None

    role_arn: t.Optional[str] = None

    external_id: t.Optional[str] = None

    endpoint_url: t.Optional[str] = None
